transcom/mymove
# B-18156: Service items address updates when shipment update

## [B-18156](https://www13.v1host.com/USTRANSCOM38/Default.aspx?menu=StoryBoardPage&feat-nav=--m2)

## Summary

This MR introduces the functionality of the DDDSIT, DDFSIT, DDASIT, and  DDSFSC MTO Service Items Address being updated when the GHC Prime updates the delivery location on the shipment. 

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the office
2. Login as a TOO
3. Navigate to Edit Shipment
4. Change the shipment details delivery location
5. Check the DB to make sure the shipments serviceItems that are listed above have the same sit Destination sit address as the shipment.


## Screenshots
![image](https://github.com/transcom/mymove/assets/136514590/719dd772-f150-420e-810a-37931e9765d6)
![image](https://github.com/transcom/mymove/assets/136514590/30b55c0c-97e7-4977-a1b5-8bacfd8919b8)
